substrate/client/network/transactions/src/lib.rs

```diff
@@ -369,7 +381,7 @@ where
 					iter::once(addr).collect(),
 				);
 				if let Err(err) = result {
-					log::error!(target: "sync", "Add reserved peer failed: {}", err);
+					log::error!(target: "transactions", "Add reserved peer failed: {}", err);
```

Comment:
`LOG_TARGET`?

Reply:
Added `LOG_TARGET` and switched back to using `sync` like it was before this PR. I used `transactions` as a temporary target to aid in debugging but forgot to revert the changes